readme.md

```diff
@@ -2,19 +2,34 @@
 
 <img src="docs/dymension.png" alt="banner" width="830"/>
 
-## The Settlement Layer of the Dymension protocol
-
 ![license](https://img.shields.io/github/license/dymensionxyz/dymension)
 ![Go](https://img.shields.io/badge/go-1.18-blue.svg)
 ![issues](https://img.shields.io/github/issues/dymensionxyz/dymension)
 ![tests](https://github.com/dymensionxyz/dymint/actions/workflows/test.yml/badge.svg?branch=main)
 ![lint](https://github.com/dymensionxyz/dymint/actions/workflows/lint.yml/badge.svg?branch=main)
 
-### Step 1: Install Go
+## Overview
+
+Welcome to Dymension Hub, the **Settlement Layer of the Dymension protocol**. 
```

Comment:
Welcome to the Dymension Hub, the **Settlement Layer of the Dymension protocol**.

- Add "the" before Dymension Hub